src/net_processing.cpp

```diff
@@ -144,6 +144,21 @@ void static ProcessGetData(CNode *pfrom, const Consensus::Params &consensusParam
                     pfrom->fDisconnect = true;
                     fSend = false;
                 }
+                // Avoid leaking prune-height by never sending blocks below the
+                // NODE_NETWORK_LIMITED threshold.
+                // Add two blocks buffer extension for possible races
+                if (send && !pfrom->fWhitelisted &&
```

Comment:
`send` -> `fSend`?

Reply:
good catch. i wonder why it didn't error in compilation if send was never defined